### PR TITLE
Using toUint96 instead of toUint64

### PR DESCRIPTION
**Pull request by josojo**
Awesome catch @fedgiac 

Creating my own library for casting, as uint96 casting is not supported by openzepplin. Later on, we should submit a PR to them.

### CODE REVIEW

File: contracts/libraries/SafeCast.sol [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+/**
+ * @dev Wrappers over Solidity's uintXX/intXX casting operators with added overflow
+ * checks.
+ *
+ * Logic was copied and modified from here: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/SafeCast.sol
+ */
+library SafeCast {
+    function toUint96(uint256 value) internal pure returns (uint96) {
+        require(value < 2**95, "SafeCast: value doesn't fit in 96 bits");
```
Comment (fedgiac) [inline contracts/libraries/SafeCast.sol]:
```suggestion
        require(value < 2**96, "SafeCast: value doesn't fit in 96 bits");
```

File: contracts/libraries/SafeCast.sol [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        return uint96(value);
+    }
+
+    function toUint64(uint256 value) internal pure returns (uint64) {
+        require(value < 2**63, "SafeCast: value doesn't fit in 64 bits");
```
Comment (fedgiac) [inline contracts/libraries/SafeCast.sol]:
```suggestion
    function toUint64(uint256 value) internal pure returns (uint64) {
        require(value < 2**64, "SafeCast: value doesn't fit in 64 bits");
```

**Review by fedgiac — COMMENTED** [review]
I think an extra bit is missing, [here](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/SafeCast.sol#L48-L51) is the link to OpenZeppelin's safe cast for comparison.

File: contracts/libraries/SafeCast.sol (hunk 1 above)
Comment (josojo) [reply contracts/libraries/SafeCast.sol]: Ups, I must have peeked at toInt , instead of toUint. sorry

**Review by fedgiac — APPROVED** [review]
:heavy_check_mark:

**Pull request merged**